tests/e2e_tests/conftest.py

```diff
@@ -72,3 +62,45 @@ def authenticator_fund_config(request):
         )
     else:
         raise ValueError(f"not configured for {e2e_env}")
+
+
+@pytest.fixture()
+def user_auth(domains, authenticator_fund_config, page):
+    email_address = generate_email_address(
+        test_name="test_submit_report",
```

Comment:
Good catch and suggestion. made the change.

`test_name` is actually `request.node.originalname`, this returns the name of the test function
`request.node.name` comes with additional suffixes like `[chronium]`, `[firefox]`.

Added the `user_roles` marker, but only for submit, as `find` test does not care for user roles.

Reply:
Probably fine for now 👍  The extra `[...]` is because of test parameterisation. In theory if we were parameterising these tests ourselves, we would probably still want to include that in the email address generated so that it's unique per test run. As it stands now, it's probably fine to use `originalname`.